addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEP.java

```diff
@@ -76,8 +77,8 @@ public EEP(ERP1Message packet) {
         setOptionalData(packet.getOptionalPayload());
     }
 
-    public EEP convertFromCommand(String channelId, String channelTypeId, Command command, State currentState,
-            Configuration config) {
+    public void convertFromCommand(String channelId, String channelTypeId, Command command,
```

Comment:
"convert" in the method name implies an input and an output and you just removed the output. Please also rename the method.

Reply:
Hmm I'd rather revert the removal of the return value. Renaming this method would lead to several other renames, because there are other methods in this unit following the same naming scheme, and one gets overridden by many child classes. Is that OK?